Ical.Net/Serialization/SerializationContext.cs

```diff
@@ -83,13 +83,13 @@ public virtual void Push(object? item)
         return null;
     }
 
-    public virtual object GetService(Type serviceType) => _mServiceProvider.GetService(serviceType);
+    public virtual object? GetService(Type serviceType) => _mServiceProvider.GetService(serviceType);
```

Comment:
@minichma This is in line with the current implementation, but I'd rather change the behavior to throw when a service cannot be found?

Reply:
Agree. Actually I think we should get rid of this whole pseudo-DI stuff. It significantly increases complexity and decreases readability while not bringing much value. But for now I agree that throwing would be better.

Reply:
@minichma Completed - not only more simple but also brings a nice increase in coverage